notebooks/06_k_mean_clustering_on_sphere.ipynb

```diff
@@ -0,0 +1,201 @@
+{
```

Comment:
<p>Nice, do you know why the sphere looks like a potato? I had the same issue</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/874/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Do you need to set the aspect ratio to equal?

Reply:
https://matplotlib.org/3.1.1/api/_as_gen/matplotlib.axes.Axes.set_aspect.html

Reply:
<p>This does not work for 3D plots but <code>ax.auto_scale_xyz([-1, 1], [-1, 1], [-1, 1])</code>  seems to do the trick</p>